test/benchmarks/netperf/Makefile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: docker nptests push
+
+DOCKERREPO       := girishkalele/netperf-latest
```

Comment:
You probably want to put it in some repo with a public read access, so anyone can use this, or use the credentials from someone who's running this. I'm not sure what's the current policy around this. @bgrant0607